MQTicketService/Dockerfile

```diff
@@ -11,37 +11,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM openjdk:8-alpine
+FROM maven:3.6-openjdk-11 AS build
 USER root
-RUN apk update && \
-    apk add wget;
 #
 COPY ./TicketGenerator /data/TicketGenerator
 #
-# Setup JAVA_HOME -- useful for docker commandline
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
-RUN export JAVA_HOME
 #
 # Compile the source code
 RUN cd /data/TicketGenerator && \
-    mkdir jarFiles && \
-    cd jarFiles && \
-    wget https://repo1.maven.org/maven2/com/ibm/mq/com.ibm.mq.allclient/9.2.0.0/com.ibm.mq.allclient-9.2.0.0.jar && \
-    wget https://repo1.maven.org/maven2/javax/jms/javax.jms-api/2.0.1/javax.jms-api-2.0.1.jar && \
-    wget https://repo1.maven.org/maven2/org/json/json/20190722/json-20190722.jar && \
-    cd /data/TicketGenerator && \
-    javac -cp /data/TicketGenerator/jarFiles/*: /data/TicketGenerator/com/ibm/mq/badge/*.java;
+    mvn -f /data/TicketGenerator/pom.xml clean package;
 #
 #
 # Switch to the MQ image
 # Tested with
-#     9.2.0.0-r2
-#     9.2.0.0-r1
-#     9.1.5.0-r2
-#     9.1.5.0-r1
+#     9.2.4.0-r1
 #
 FROM ibmcom/mq:latest
 USER 0
+
+RUN curl -Lo OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jre_x64_linux_hotspot_11.0.13_8.tar.gz
```

Comment:
These 3 RUN steps should be collapsed into a single RUN with && separators.

Reply:
Updated as required.